packages/relay-service/lib/index.js

```diff
@@ -80,8 +80,20 @@ const getMessages = async ({keyPairDocs, limit = 20}) => {
         const keyAgreementKey = await getDerivedAgreementKey(keyPairDoc);
         let jwe = item.msg;
 
-        if (isBase64(jwe)) {
-          jwe = fromBase64(jwe);
+        if (typeof jwe === 'string') {
+          // TODO: check for JWT in future here
+
+          try {
+            if (isBase64(jwe)) {
+              jwe = fromBase64(jwe);
+            } else {
+              jwe = JSON.parse(jwe);
+            }
+          } catch (e) {
+            // Not valid JSON but still a string, show some kind of error?
```

Comment:
For now will skip this message, maybe could create a sentry error

Reply:
debug log could work for now, better than nothing